src/tests/helpers/index.ts

```diff
@@ -8,6 +8,7 @@ export interface ConversationTurn<T> {
 
 export interface TestCase<T, G> {
   description: string,
+  slots: Slot<G>[],
   abilities: Ability<T, G>[],
```

Comment:
Parameter `slots` and `abilities` can be simplified to pass in `flow` which contains both. Simplifies parameter from 2 -> 1